test/controllers/to_do_items_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class ToDoItemsControllerTest < ActionController::TestCase
+  fixtures :items
+
+  test "should create empty item in concrete list" do
+    get :new, id: items(:first).id
+    item = FactoryGirl.build(:only_with_list_reference)
+
+    assert_response :success
```

Comment:
@vlasiak, What about the following name: `should instantiate a new item in the list`?